Automattic/wp-calypso
# Update courses/webinars header padding

Related to #

## Proposed Changes

* Update happy blocks search header padding for wpsupport3 courses and webinar pages

## Testing Instructions



* Checkout branch
* Sync happyblocks with sandbox
* Visit courses / webinars page to verify styling changes

## Pre-merge Checklist
